fetch.bs

```diff
@@ -4017,9 +4032,10 @@ steps:
 
  <li><p>Set <var>response</var>'s <a for="response">timing info</a> to <var>timingInfo</var>.
 
- <li><p><a href="https://github.com/w3c/resource-timing/pull/261">Mark resource timing</a> for
- <var>timingInfo</var>, <var>originalURL</var>, <var>initiatorType</var>, and <var>global</var>.
- <!-- TODO -->
+ <li><p>
+ <a href="https://w3c.github.io/resource-timing/#dfn-mark-resource-timing">Mark resource timing</a>
```

Comment:
Why is this needed, is it not exported? Let's export it first.

Reply:
Sure, waiting for Resource-Timing to be published to TR (bikeshed vs. Respec stuff).

Reply:
Removing this bit for now, will update once the refs are resolvable.